src/sentry/api/endpoints/sentry_app_interaction.py

```diff
@@ -48,14 +48,15 @@ def get(self, request, sentry_app):
             }
         )
 
-    def post(self, request, sentry_app):
+    def post(self, request, sentry_app, **kwargs):
         """
         Increment a TSDB metric relating to Sentry App interactions
 
         :param string tsdbField         the name of the TSDB model to increment
         :param string componentType     required for 'sentry_app_component_interacted' metric
         """
         # Request should have identifier field stored in TSDBModel
+        key = None
```

Comment:
This was just to fix the error from the `reportUnboundVariable` from the [new commit hook](https://github.com/getsentry/sentry/blob/master/pyrightconfig-commithook.json) -not a change related to this PR really, but since I touched the file I had to update it.